README.md

```diff
@@ -1,19 +1,8 @@
 # Sample Auth implementations 
 
-## Javascript 
-### Setup
-This setup assumes that your machine has `nodejs` (with `npm`) installed. `cd` into the root directory, then run `npm i` to install the dependencies.
-
-### Configuration
-You'll need to set the environment variables `CLIENT_ID`, `CLIENT_SECRET` and `REDIRECT_URI` to match the application configuration in Atlas. You'll most likely want to point `ngrok`(https://ngrok.com/) at this local server, and use the HTTPS version as your redirect URI.
-
-If using ngrok, you'll want to run `ngrok http 3000`, and set the `REDIRECT_URI` in both the Atlas application config and the environment variable to match the HTTPS address generated by ngrok.
-
-### Running the application
-
-After the dependencies have finished installing, run the below block to start.
-```
-DEBUG="myapp:*" CLIENT_ID = "..." CLIENT_SECRET = "..." REDIRECT_URI = "https://....ngrok.io/callback" npm start
-```
-
-To kick off a journey, go to http://localhost:3000/redirect
\ No newline at end of file
+## Setup
+1. Setup and launch `ngrok` on port `3000` to create a redirect URI, set this URI as `REDIRECT_URI` in the `Dockerfile` for the implementation you intend to run.
+2. Input your application `CLIENT_ID` and `CLIENT_SECRET` into the same `Dockerfile`.
+3. Run `make`
+4. Launch the example implementation with `docker run auth-sample-[language]`. <br>E.g: `docker run auth-sample-go`
```

Comment:
`<br>` isn't a thing in markdown. Also, the ultimate nit: Latin doesn't use upper case at the start of a sentence.

Reply:
but this is English?

Reply:
`e.g.` isn't